test/Driver/linker-order.ll

```diff
@@ -0,0 +1,9 @@
+; RUN: llc -mtriple armv7--linux-gnueabihf -filetype obj -o - %s | swift-autolink-extract -o - - | FileCheck %s
```

Comment:
Done.  Added a comment explaining the test.  I guess I never got into the habit of that working on LLVM.